frontend/src/components/maps/leaflet/InfoSlideOut/ParcelAttributes.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { ListGroup } from 'react-bootstrap';
+import { IParcel } from 'actions/parcelsActions';
+import { Label } from 'components/common/Label';
+import './InfoSlideOut.scss';
+import { formatMoney } from 'utils/numberFormatUtils';
+
+interface IParcelAttributes {
+  /** the selected parcel information */
+  parcelInfo: IParcel | null;
+}
+
+/**
+ * Displays parcel specific information needed on the information slide out
+ * @param parcelInfo the selected parcel data
+ */
+export const ParcelAttributes: React.FC<IParcelAttributes> = ({ parcelInfo }) => {
+  const formatAssessed = parcelInfo?.assessed
+    ? formatMoney(parcelInfo?.assessed)
+    : formatMoney(parcelInfo?.evaluations[0].value);
+  return (
+    <>
+      <ListGroup>
+        <Label className="header">Parcel attributes</Label>
+        <ListGroup.Item>
+          <Label>Lot size:</Label>
+          {parcelInfo?.landArea + ' hectares'}
+        </ListGroup.Item>
+      </ListGroup>
+      <ListGroup>
+        <Label className="header">Legal description</Label>
+        <ListGroup.Item>{parcelInfo?.landLegalDescription}</ListGroup.Item>
+      </ListGroup>
+      <ListGroup>
+        <Label className="header">Valuation</Label>
+        <ListGroup.Item>
+          <Label>Assessed value:</Label>
+          {formatAssessed}
+        </ListGroup.Item>
+      </ListGroup>
+      {parcelInfo?.buildings && (
+        <ListGroup>
+          <Label className="header">Associated Buildings</Label>
```

Comment:
I'm displaying only three buildings and then "+ __ more" because some parcels (like Uvic) have upwards of 60 buildings